commanderbot_ext/vote/__init__.py

```diff
@@ -0,0 +1,6 @@
+from discord.ext import commands
+from commanderbot_ext.vote.vote_cog import VoteCog
+
+
+def setup(bot: commands.Bot):
+    bot.add_cog(VoteCog(bot))
```

Comment:
Make sure you run the auto-formatter before committing. There should be a newline at the end of every file.

Reply:
Fixed in latest commit